src/Compilers/CSharp/CscCore/csc.deps.json

```diff
@@ -0,0 +1,135 @@
+{
```

Comment:
Why is this file getting checked in?  Not familiar with it but at a glance looks like a lock file.